services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/config/JdbcProperties.java

```diff
@@ -38,6 +38,9 @@ public class JdbcProperties {
     private String username;
     private String password;
     private Integer maximumPoolSize;
+    private Integer minimumPoolSize;
```

Comment:
now that we always have a (default) value for these properties, we should probably define them as `int`s instead of `Integer`s, shouldn't we?